packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -824,8 +823,12 @@ export class InputManager {
                 if (!this._meshPickProceed && ((AbstractActionManager && AbstractActionManager.HasTriggers) || scene.onPointerObservable.hasObservers())) {
                     this._initActionManager(null, clickInfo);
                 }
-                if (!pickResult) {
-                    pickResult = this._currentPickResult;
+
+                if (scene.skipPointerUpPicking) {
+                    pickResult = new PickingInfo();

```

Comment:
would it make sense to store a static version of an empty pickingInfo instead of creating an empty one every time? (might not make sense, it's more of a question :-))

Reply:
A static version might make more sense for skipping.  I'm not 100% if anything is changed within the pickResult object as it passes through `_processPointerUp` and `_processPointerDown` (specifically when `step.action` is called) so I'd have to investigate that first before implementing it.